aws_security_group.tf

```diff
@@ -112,296 +112,296 @@ resource "aws_security_group" "acme_web" {
     from_port   = 22
     to_port     = 22
     protocol    = "tcp"
-    cidr_blocks = ["0.0.0.0/0","19.16.0.0/24"]
+    cidr_blocks = ["<cidr>", "<cidr>"]
   }
 
   ingress {
     from_port   = 23
     to_port     = 23
     protocol    = "tcp"
-    cidr_blocks = ["0.0.0.0/0","19.16.0.0/24"]
+    cidr_blocks = ["<cidr>", "<cidr>"]
   }
 
   ingress {
     from_port   = 25
     to_port     = 25
     protocol    = "tcp"
-    cidr_blocks = ["0.0.0.0/0","19.16.0.0/24"]
+    cidr_blocks = ["<cidr>", "<cidr>"]
   }
 
   # HTTP access from the VPC
   ingress {
     from_port   = 80
     to_port     = 80
     protocol    = "tcp"
-    cidr_blocks = ["0.0.0.0/0", "192.164.0.0/24"]
+    cidr_blocks = ["<cidr>", "192.164.0.0/24"]
   }
 
   # HTTPS access from the VPC
   ingress {
     from_port   = 443
     to_port     = 443
     protocol    = "tcp"
-    cidr_blocks = ["0.0.0.0/0", "192.164.0.0/24"]
+    cidr_blocks = ["<cidr>", "192.164.0.0/24"]
   }
 
   ingress {
     from_port   = 445
     to_port     = 445
     protocol    = "tcp"
-    cidr_blocks = ["0.0.0.0/0", "192.164.0.0/24"]
+    cidr_blocks = ["<cidr>", "<cidr>"]
   }
 
   ingress {
     from_port   = 4505
     to_port     = 4505
     protocol    = "tcp"
-    cidr_blocks = ["0.0.0.0/0", "192.164.0.0/24"]
+    cidr_blocks = ["<cidr>", "<cidr>"]
   }
 
   ingress {
     to_port     = 4506
     from_port   = 4506
     protocol    = "-1"
-    cidr_blocks = ["0.0.0.0/0", "192.164.0.0/24"]
+    cidr_blocks = ["<cidr>", "<cidr>"]
   }
 
   ingress {
     to_port     = 3020
     from_port   = 3020
     protocol    = "tcp"
-    cidr_blocks = ["0.0.0.0/0", "192.164.0.0/24"]
+    cidr_blocks = ["<cidr>", "<cidr>"]
   }
 
   ingress {
     to_port     = 61621
     from_port   = 61621
     protocol    = "-1"
-    cidr_blocks = ["0.0.0.0/0", "192.164.0.0/24"]
+    cidr_blocks = ["<cidr>", "<cidr>"]
   }
 
   ingress {
     to_port     = 7001
```

Comment:
**Accurics remediation**
 
**[HIGH]: Security Groups - Unrestricted Specific Ports - MSSQL Debugger(TCP),135**
 
**Vulnerability**
Security Groups - Unrestricted Specific Ports - MSSQL Debugger(TCP),135
 
**Remediation**
Limit the access scope for 'MSSQL Debugger' to only allow access in internal networks and limited scope. If public interface exists, remove it and limit the access scope within the VPC only to applications or instances that requires access. Amazon Reference: http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/authorizing-access-to-an-instance.html. As a further protection step, Use 'Tamper Protection' (Full Protection mode) and region lock to prevent unauthorized changes to network access settings, refer to /360003963194#Set_an_AWS_Security_Group_to_Full_Protection for more details.

source: accurics-